packages/server/src/notification/desktop-notif.entity.ts

```diff
@@ -33,4 +32,10 @@ export class DesktopNotifModel extends BaseEntity {
 
   @Column({ nullable: true })
   userId: number;
+
+  @CreateDateColumn({ type: 'timestamp' })
+  createdAt: Date;
+
+  @Column({ type: 'text', nullable: true })
+  name: string;
```

Comment:
Added a name field. When someone subscribes, the frontend will also send over a name like "Firefox on Ubuntu"